users/management/commands/retire_users.py

```diff
@@ -63,12 +69,21 @@ def add_arguments(self, parser):
             help="Single or multiple username(s) or email(s)",
         )
 
+        parser.add_argument(
+            "-b",
+            "--block",
+            action="store_true",
+            dest="block_users",
+            help="If provided, user's email will be hashed and marked block",
```

Comment:
@asadiqbal08 Wording suggestion:
`If provided, user's email will be hashed and added to the blocklist`